### ISSUE TITLE
Flag differences between scans

### ISSUE THREAD

**Issue by tinkerspy** [angryip/ipscan#362]
**Is your feature request related to a problem? Please describe.**
I use angryip to find new devices on my network after I plugged them in.

**Describe the solution you'd like**
A way to tell the difference between two scans. 
Flag the hosts that were not detected in scan A but are detected in scan B. 
For instance by displaying the IP number in green.

**Describe alternatives you've considered**

**Additional context**

**Comment by KDMcMullan** [comment]
Yes. This is exactly the request I was about to make. I'd also suggest that on a subsequent scan, changes might be highlighted. This would be particularly useful if a device has been added between scans and we hope to learn its IP address.
Thanks.
